components/central-application-connectivity-validator/cmd/centralapplicationconnectivityvalidator/options.go

```diff
@@ -117,5 +112,8 @@ func (o *options) validate() error {
 	if !strings.Contains(o.appRegistryPathPrefix, o.appNamePlaceholder) {
 		return fmt.Errorf("appRegistryPathPrefix '%s' should contain appNamePlaceholder '%s'", o.appRegistryPathPrefix, o.appNamePlaceholder)
 	}
+	if o.syncPeriod > time.Duration(o.cacheExpirationSeconds)*time.Second {
```

Comment:
I'd be super nice to add it also to the options' description in the `*.md` file and at the beginning of the `parseOptions()` function so an user knows about this validation from the start

Reply:
In the readme there is a section "Local cache refresh" which describes this restriction